readme.txt

```diff
@@ -1,64 +1,91 @@
+Tags: Jetpack, WordPress.com, backup, security, related posts, CDN, speed, anti spam, social sharing, SEO, video, stats
 Stable tag: 4.5
 Requires at least: 4.6
 Tested up to: 4.7
 
-Increase your traffic, view your stats, speed up your site, and protect yourself from hackers with Jetpack.
+The one plugin you need for stats, related posts, search engine optimization, social sharing, protection, backups, faster content, and email lists.
 
 == Description ==
 
-Jetpack simplifies managing WordPress sites by giving you visitor stats, security services, speeding up images, and helping you get more traffic. Jetpack is a free plugin.
+Whatever sort of site you’re running, Jetpack’s features are designed to keep your site secure, increase traffic, and engage your readers.
 
-= Traffic Growth & Insights =
-Jetpack tells you how many visits your site gets and helps you **get more traffic** with tools like Related Posts, Publicize, Enhanced Distribution, and Sharing.
+= Traffic and SEO Tools =
+Traffic is the lifeblood of any website. Jetpack includes:
 
-= Security =
-Jetpack **protects your site** against brute force attacks and unauthorised logins. We also monitor your site for downtime and keep your plugins updated.
+* [free] site stats and analytics
+* [free] automatic promotion and sharing on Facebook, Twitter, LinkedIn, and WordPress.com
+* [free] related posts
+* [paid] search engine optimization tools for Google, Bing, Twitter, Facebook, and WordPress.com
+* [paid] advert programme that includes AdSense, WordAds, Facebook Ads, AOL, Amazon, Google AdX, and Yahoo
 
-= Image Performance =
-Jetpack automatically **optimizes and speeds up** images using the global WordPress.com content delivery network. This saves you hosting costs by reducing bandwidth.
+= Security and Backup Services =
+Stop worrying about data loss, downtime, and hacking. Jetpack provides: 
 
-= Centralized Management =
-With Jetpack comes a **centralized dashboard** on WordPress.com. Manage plugins and menus, publish posts, and view enhanced site stats for all of your sites.
+* [free] brute force attack protection
+* [free] downtime and uptime monitoring
+* [free] secure and two-factor authentication
+* [paid] malware scanning, code scanning, and threat resolution
+* [paid] site backups, restores, and migrations
 
-= A few more things that our users love =
-Jetpack includes other features that help you customize your site including Custom CSS, Contact Forms, Galleries and Carousels, Notifications and Subscriptions, Configurable Widgets, and [many more](https://jetpack.com/features).
+= Writing and Content Creation =
+Jetpack helps your writing and image creation processes with:
 
-= Dedicated Support =
+* [free] a high-speed CDN for your images
+* [free] carousels, slideshows, and tiled galleries
+* [free] embed media from YouTube, Google Documents, Spotify and more
+* [free] sidebar customization including Facebook feeds, Twitter feeds, and RSS feeds

```

Comment:
`sidebar customization including Facebook, Twitter, RSS, Author, and Blog Stats widgets`